main.go

```diff
@@ -173,16 +172,14 @@ func _main(ctx context.Context) error {
 			DBName:       cfg.PGDBName,
 			QueryTimeout: cfg.PGQueryTimeout,
 		}
+
 		// Replace {netname} with curnet
-		if strings.Contains(dbi.DBName, "{netname}") {
-		  reg, err := regexp.Compile("[^a-zA-Z]+")
-		  if err != nil {
-		    return err
-		  }
-		  
-		  var networkName = reg.ReplaceAllString(curnet.String(), "");
-		  dbi.DBName = strings.Replace(dbi.DBName, "{netname}", networkName, 1);
+		const nettemplate = "{netname}"
+		if strings.Contains(dbi.DBName, nettemplate) {
+			networkname := strings.ToLower(curnet.String())
+			dbi.DBName = strings.Replace(dbi.DBName, nettemplate, networkname, -1)
```

Comment:
Basically something like this:

```go
		// Replace {netname} with active network name.
		const netTemplate = "{netname}"
		if strings.Contains(dbi.DBName, netTemplate) {
			dbi.DBName = strings.Replace(dbi.DBName, netTemplate,
				netName(activeChain), -1)
		}
```

Reply:
But now that I think about this, the `Contains` check can be omitted:

```go
dbi.DBName = strings.Replace(dbi.DBName, netTemplate, netName(activeChain), -1)
```

Reply:
Okay, I'm about to commit an update for this.  It's a one line change with a comment now.  I removed the constant because it's only used in one place now, but I can easily add it back in.  I installed Atom and set it up with a linter and everything.  So far so good, except it changed a few files automatically so I reset them.  Thanks for all your feed back.